packages/react-components/src/components/knowledge-graph/styles.scss

```diff
@@ -0,0 +1,16 @@
+.knowledgeGraph-container {
+  height: 100%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+.search-bar {
+  min-width: 500px;
+  flex: 0;
+  margin: 10px;
+}
+.action-buttons {
+  min-width: 300px;
```

Comment:
This should use `rem`.

Reply:
same here, this is a small div/component in the KG UI the other div like canvas search bar .. etc all use px adjusting this alone means as the default value of 16px grow or shrink this is the only div that will grow and shrink while the other stay the same (coz the are using px). by the way this one is also not part of the main purpose of the PR they were inline styling just moved them to stylesheet file. I am not opposed to rem but I think partial rem usage while many other inter related div use px is dangerous. I think we should create an asana task so this will be a comprehensive task that move everything to rem (or use both) with extensive testing. And if i go down this rabbit hole :) the scope of this PR will be very big. I can see the change will be straight forward if its a font-size styling but I am bit reluctant to do this  partially for width, height ..etc.

Great discussion [here ](https://stackoverflow.com/questions/11799236/should-i-use-px-or-rem-value-units-in-my-css)about using rem vs px, but in general I am not opposed to using rem but it should be part of comprehensive migration task with careful testing.